root/layout/components/Footer.js

```diff
@@ -42,7 +42,7 @@ const Footer = ({$c, ...props}) => {
 
         {DBDefs.GIT_BRANCH ? (
           <>
-            <br />
+            <br key="git_details_br" />
```

Comment:
Do these need keys? They don't appear in an array and I don't get any errors logged for them.

Reply:
Oops, that was before before https://github.com/metabrainz/musicbrainz-server/commit/9312de9a07d8b7aab8c1fa539c66987ba7768598. Removed from https://github.com/metabrainz/musicbrainz-server/pull/790/commits/9c8a90a688a2fda95ed746bc4e7b36d0137e6c0b.